backend/src/test/java/com/ody/mate/service/MateServiceTest.java

```diff
@@ -172,4 +174,31 @@ void saveMateWithDuplicateMember() {
                     .isInstanceOf(OdyBadRequestException.class);
         }
     }
+
+    @DisplayName("참여자 삭제 시, 구독하고 있는 fcmTopic 취소힌다.")
+    @Test
+    void unSubscribeTopicWhenDeleteMate() {
+        Mate mate = fixtureGenerator.generateMate();
+        FcmTopic fcmTopic = new FcmTopic(mate.getMeeting());
+        DeviceToken deviceToken = mate.getMember().getDeviceToken();
+
+        mateService.delete(mate);
+
+        Mockito.verify(fcmSubscriber, Mockito.times(1)).unSubscribeTopic(fcmTopic, deviceToken);
+    }
+
+    @DisplayName("회원 삭제 시, 구독하고 있는 모든 fcmTopic을 취소한다.")
+    @Test
+    void unSubscribeAllTopicsWhenDeleteMember() {
+        Member jojo = fixtureGenerator.generateMember("jojo");
+        Member olive = fixtureGenerator.generateMember("olive");
+
+        fixtureGenerator.generateMate(jojo);
+        fixtureGenerator.generateMate(jojo);
+        fixtureGenerator.generateMate(olive);
+
+        mateService.deleteAllByMember(jojo);
+
+        Mockito.verify(fcmSubscriber, Mockito.times(2)).unSubscribeTopic(any(), any());
```

Comment:
지금도 좋지만 타입 안전성 정도는 같이 체크해주어도 좋을 것 같아요!
```suggestion
        Mockito.verify(fcmSubscriber, Mockito.times(2)).unSubscribeTopic(any(Meeting.class), any(DeviceToken.class));
```

Reply:
좋아요~ 반영완!